README.md

```diff
@@ -225,4 +227,4 @@ emit "Once upon a time, at the foot of a great mountain ..." ^
 [pack]: https://binref.github.io/#refinery.pack
 
 [jupyter]: https://jupyter.org/install
-[jupyter-vscode]: https://code.visualstudio.com/docs/datascience/jupyter-notebooks
\ No newline at end of file
+[jupyter-vscode]: https://code.visualstudio.com/docs/datascience/jupyter-notebooks
```

Comment:
It looks like my editor killed a trailing whitespace here; I'll keep this as part of the patch for now, but can remove it if you'd rather keep PRs focused on single changes.

Reply:
Nah, this is perfectly fine. Thanks for the PR!